### ISSUE TITLE
Volume control with TouchBar don't work

### ISSUE THREAD

**Issue by Erik262** [MonitorControl/MonitorControl#844]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

I can't change the volume through my TouchBar on my MacBook, but the volume slider of MonitorControl works. The speaker symbols for volume and mute are greyed out, but the mute symbol still works even it is colored grey.

<img width="1085" alt="Touch Bar Shot 2021-12-13 at 22 05 40" src="https://user-images.githubusercontent.com/11977647/145889026-108b2fd9-64c0-4a63-a00f-f6813f448c31.png">

### Steps to reproduce

Try to adjust the volume via TouchBar speaker symbols on your TouchBar.

### Expected behavior

Adjusted volume through MacBook TouchBar

### Anything else?

Nope. :)

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.1
- Mac model: MacBook Pro (13-inch, 2016, Four Thunderbolt 3 Ports)
- MonitorControl version: 4.0.2 Build 6965
- Monitor(s): 1
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
hi @Erik262 - to me it seems like the mouse keys do work (base don the gif you sent). Or am I missing something? :)

**Comment by Erik262** [comment]
@waydabber Yes this works, have to do it manually via the slider in the app.

**Comment by waydabber** [comment]
Hi, just saw that what I wrote "mouse keys work" does not make sense lol - I meant "touchbar keys work". So it seems you are using the key up/down button on the touch bar and the volume is controlled in your GIF.

**Comment by Erik262** [comment]
@waydabber, haha I also thought, what is this guy meaning xD.  Yes, the touch bar is completely working, and the volume buttons on the touch bar as well when the computer is not plugged into the monitor.

By tapping on the volume button, there should be a volume slider that should look like this. (In this example I connected my Headphones)
<img width="1085" alt="Touch Bar Shot 2021-12-14 at 18 16 46" src="https://user-images.githubusercontent.com/11977647/146047338-3ee06017-ee43-473a-9a52-6c3437b300ed.png">
